src/main/java/org/folio/consortia/config/kafka/KafkaConsumerInterceptor.java

```diff
@@ -0,0 +1,68 @@
+package org.folio.consortia.config.kafka;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.kafka.clients.consumer.ConsumerInterceptor;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.common.header.Header;
+import org.apache.kafka.common.header.Headers;
+import org.folio.spring.DefaultFolioExecutionContext;
+import org.folio.spring.FolioModuleMetadata;
+import org.folio.spring.integration.XOkapiHeaders;
+import org.folio.spring.scope.FolioExecutionScopeExecutionContextManager;
+
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+public class KafkaConsumerInterceptor implements ConsumerInterceptor<Object, Object> {
```

Comment:
We don't need kafka consumer interceptors any more, please reffer to this PR:
 https://github.com/folio-org/mod-data-export-spring/pull/240/files